src/contracts/pods/EigenPodManager.sol

```diff
@@ -123,16 +123,16 @@ contract EigenPodManager is
         // skip making a call to the DelegationManager if there is no change in delegateable shares
```

Comment:
we need to scale inside of `_calculateChangeInDelegatableShares`

Reply:
and update podOwnerShares after

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/695/commits/2a8d0eb2f873c733791ba1e3f5b3ac8eda3e79b7
https://github.com/Layr-Labs/eigenlayer-contracts/pull/695/commits/c54e9c7c6e808ecdbdf34d6400dd29bbcd326b34
renamed to indicate shares